adafruit_emc2101/__init__.py

```diff
@@ -169,57 +157,80 @@ class EMC2101:  # pylint: disable=too-many-instance-attributes
             emc.manual_fan_speed = 25
 
 
-
-
     If you need control over PWM frequency and the controller's built in temperature/speed
     look-up table (LUT), you will need :class:`emc2101_lut.EMC2101_LUT` which extends this
     class to add those features, at the cost of increased memory usage.
+
+    Datasheet: https://ww1.microchip.com/downloads/en/DeviceDoc/2101.pdf
     """
 
-    _part_id = ROUnaryStruct(_REG_PARTID, "<B")
-    _mfg_id = ROUnaryStruct(_REG_MFGID, "<B")
-    _int_temp = ROUnaryStruct(_INTERNAL_TEMP, "<b")
-
-    # Some of these registers are defined as two halves because
-    # the chip does not support multi-byte reads or writes, and there
-    # is currently no way to tell Struct to do a transaction for each byte.
-
-    # IMPORTANT! the sign bit for the external temp is in the msbyte so mark it as signed
-    # and lsb as unsigned
-    _ext_temp_msb = ROUnaryStruct(_EXTERNAL_TEMP_MSB, "<b")
-    _ext_temp_lsb = ROUnaryStruct(_EXTERNAL_TEMP_LSB, "<B")
-
-    _tach_read_lsb = ROUnaryStruct(_TACH_LSB, "<B")
-    _tach_read_msb = ROUnaryStruct(_TACH_MSB, "<B")
-    _tach_mode_enable = RWBit(_REG_CONFIG, 2)
-    _tach_limit_lsb = UnaryStruct(_TACH_LIMIT_LSB, "<B")
-    _tach_limit_msb = UnaryStruct(_TACH_LIMIT_MSB, "<B")
-    # temp used to override current external temp measurement
-    forced_ext_temp = UnaryStruct(_TEMP_FORCE, "<b")
-    """The value that the external temperature will be forced to read when `forced_temp_enabled` is
-    set. This can be used to test the behavior of the LUT without real temperature changes"""
-    forced_temp_enabled = RWBit(_FAN_CONFIG, 6)
-    """When True, the external temperature measurement will always be read as the value in
-    `forced_ext_temp`"""
-
-    _fan_setting = UnaryStruct(_REG_FAN_SETTING, "<B")
-    _pwm_freq = RWBits(5, _PWM_FREQ, 0)
-    _fan_lut_prog = RWBit(_FAN_CONFIG, 5)
-    invert_fan_output = RWBit(_FAN_CONFIG, 4)
```

Comment:
It looks like this was exposed, if we removing it, it;s technically breaking.  I'd recommend adding back in if there's no explicit reason to remove it.

Reply:
ok, I'll move back. I was kind-of assuming that the reason for the _ prefix to these names was to remind users that they were internal, implementation-specific variables and thus not to be relied on.
However, I can move them if you wish.

Reply:
That's it's use, but it looks like it was removed entirely and was previously exposed, so it's possible someone was relying on it because it's exposed, but would now have their code break.  Readding it will fix it though!

Reply:
This is in reference to `invert_fan_output`

Reply:
I have reinstated invert_fan_output now.